requirements.txt

```
@@ -1,3 +1,3 @@
 # mail_inline_css
 premailer
-extract-msg
+extract-msg<=0.27.5
```

I just noticed that the last pipeline that ran "green" was using v0.27.4: https://travis-ci.org/github/OCA/social/jobs/731915721#L481
```suggestion
extract-msg<=0.27.4
```
Sorry for the misunderstanding